third_party/amd/backend/compiler.py

```diff
@@ -36,6 +36,14 @@ def get_gfx9_limits(lhsType, rhsType):
     return lambda lhsType, rhsType: (1, 1, 1) if is_fma_supported(lhsType.scalar, rhsType.scalar) else (16, 16, 16)
 
 
+def is_pingpong_enabled(arch):
+    if arch == 'gfx942':
```

Comment:
Should we include `gfx940` and `gfx941` too?

Reply:
I thought about it but this is related to memory system performance and we haven't tested it on 940/941. Shall I file a ticket to test it on those machines and add it to this list?

Reply:
I suspect it should apply given the majority is actually instruction scheduling. But fine to keep as-is to be conservative.